### PR TITLE
Add avatar component

### PR DESCRIPTION
**Pull request by coryschadt**

### CODE REVIEW

File: components/avatar/index.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+export default ({
```
Comment (staydecent) [inline components/avatar/index.js]: Anonymous components should be avoided as they show up as `<undefined ... />` in debuggers and tests. You can use implicitly named arrow functions however:

```jsx
const Avatar = () => /* ... */
export default Avatar
```

Or you could use an explcitly named function:

```jsx
export default function Avatar () {
  /* ... */
}
```

File: components/avatar/index.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  size = '100'
+}) =>
+  <div style={`font-size: ${size}%;`} className={`avatar-wrap ${className}`}>
+    <div className={`avatar ${src && 'hasImage'}`}>
+      {src
+        ? <img src={src} alt={fullName} />
+        : <div className='initial'>{fullName[0]}</div>
+      }
+    </div>
+  </div>
```
Comment (staydecent) [inline components/avatar/index.js:17]: I think you forgot the `import './style.less'`

File: components/avatar/package.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
+{
+  "name": "@app-elements/avatar",
+  "version": "1.0.0",
+  "description": "",
```
Comment (staydecent) [inline components/avatar/package.json]: Could you add a short description?

**Review by staydecent — CHANGES_REQUESTED** [review]
Needs a few changes, but this will be a great addition to the repo!

**Comment by coryschadt** [comment]
Hey @staydecent,  Thanks for reviewing.

I have made these changes. Can you review and let me know?

Cheers

**Review by staydecent — APPROVED** [review]
💎

**Pull request merged**